pkg/trace/api/api.go

```diff
@@ -128,6 +128,7 @@ func (r *HTTPReceiver) Start() {
 	mux.HandleFunc("/v0.3/services", r.handleWithVersion(v03, r.handleServices))
 	mux.HandleFunc("/v0.4/traces", r.handleWithVersion(v04, r.handleTraces))
 	mux.HandleFunc("/v0.4/services", r.handleWithVersion(v04, r.handleServices))
+	mux.Handle("/proxy/profile", r.profileProxyHandler())
```

Comment:
In anticipation of whatever-could-happen in the future, what about versioning this endpoint too?